File: modul 3/c/PRAK305-2410817110008-MuhammadAzmaAlFaqih.c
#include <stdio.h>

int main(){
  int inputDetik, jam, menit, detik, hari;
  scanf("%d", &inputDetik);
  hari = inputDetik / 86400;
  inputDetik -= hari * 86400;
  jam = inputDetik / 3600;
  menit = (inputDetik%3600/60);
  detik = (inputDetik % 60);
  if (hari == 0){
    printf("%02d:%02d:%02d", jam, menit, detik);
  } else {
    printf("%d hari %02d:%02d:%02d", hari, jam, menit, detik);
  }
  printf("\n");
}

File: modul 3/python/PRAK305-2410817110008-MuhammadAzmaAlFaqih.py
inputDetik = int(input())
hari = inputDetik // 86400
inputDetik -= hari * 86400
jam = inputDetik // 3600
menit = (inputDetik % 3600)//60
detik = inputDetik % 60
if hari == 0:
    print("%02d:%02d:%02d" % (jam, menit, detik))
else:
    print("%d hari %02d:%02d:%02d" % (hari, jam, menit, detik))
